recipes/pasta/0.2/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+out_dir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+pasta_tools_dev_dir=$PREFIX/share
+share_dir=$PREFIX/share
+
+mkdir -p $out_dir
+mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/pasta
+
+cp -R ./* $out_dir
+
+unamestr=`uname`
+if [ $unamestr == 'Linux' ];
+then
+    wget https://github.com/smirarab/sate-tools-linux/archive/master.tar.gz
```

Comment:
Ah, I meant to change that - thanks for pointing this out.  Actually, these separate repositories of tools are required to compile the Pasta package, but I meant to use an actual rev instead of the master branch tip.  I'll fix this shortly.  Thanks again!